.github/workflows/zoningtaxlots_build.yml

```diff
@@ -65,8 +69,8 @@ jobs:
       - name: Dataloading
         run: python -m dcpy.lifecycle.builds.load load
 
-      - name: 2. build ...
-        run: ./ztl.sh build
```

Comment:
if we're scrapping calls `ztl.sh` let's also remove it. I'm good with that, it's not much of a wrapper and doesn't add much simplicity